### ISSUE TITLE
Pull Properties from TypeDefinition

### ISSUE THREAD

**Issue by araisch** [FreeOpcUa/opcua-modeler#102]
1. Add Variable
2. Change "HasTypeDefinition" to e.g. i=2368 (AnalogItemType)
3. Properties of the Type aren't pulled, neither Mandatory nor Optional ones.
![image](https://user-images.githubusercontent.com/35069589/93885398-61b1b280-fce4-11ea-9e30-1c611ccb14c0.png)
4. Should have the mandatory properties and the inherited ones and an option to choose from the optional ones

Or is there another way to reach that?

**Comment by Psoke** [comment]
THIS is exactly what I have been wondering also. In UA Modeler it simply pulls EURange with it when defining it as AnalogItemType. Can't get it to work here and its highly frustrating. 
Also why can you add a objectType when adding an object but not add a variableType when adding a variable.

**Comment by zerox1212** [comment]
> THIS is exactly what I have been wondering also. In UA Modeler it simply pulls EURange with it when defining it as AnalogItemType. Can't get it to work here and its highly frustrating.
> Also why can you add a objectType when adding an object but not add a variableType when adding a variable.

Add this functionality and send a pull request.

**Comment by araisch** [comment]
> > THIS is exactly what I have been wondering also. In UA Modeler it simply pulls EURange with it when defining it as AnalogItemType. Can't get it to work here and its highly frustrating.
> > Also why can you add a objectType when adding an object but not add a variableType when adding a variable.
> 
> Add this functionality and send a pull request.

@zerox1212 You're right: This is open source and instead of complaining aggressively anyone could fix it or add a functionality. But not everyone has the skills to do that. 
@Psoke if you just searching for a Modeler "For Free" try SIOME from Siemens. There this feature works..
